src/items/AppItem.js

```diff
@@ -42,10 +42,7 @@ class AppItem extends Component {
     url: null,
   };
 
-  constructor(props) {
-    super(props);
-    this.iframeRef = React.createRef();
-  }
+  iframeRef = React.createRef();
```

Comment:
Can you please add a comment regarding why we need this just for posterity? Thanks!